### PR TITLE
Fix/members details error

### PR DESCRIPTION
**Pull request by dabick14**

### CODE REVIEW

File: web-react-ts/src/global-utils.ts [hunk 1]
```diff
@@ -195,44 +195,68 @@ export const debounce = (callback: () => void, delay = 500) => {
...
-  titleConnection: {
-    edges: {
-      node: { title: 'Pastor' | 'Reverend' | 'Bishop' }
-    }[]
-  }
+  // titleConnection: {
+  //   edges: {
+  //     node: { title: 'Pastor' | 'Reverend' | 'Bishop' }
+  //   }[]
+  // }
```
Comment (ahadziii) [inline web-react-ts/src/global-utils.ts:202]: Is this supposed to be here ??

Comment (dabick14) [reply web-react-ts/src/global-utils.ts:202]: Well. We may want to use it later. So I'm uncommenting it

File: api/package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "@apollo/client": "^3.2.5",
-    "@neo4j/graphql": "^3.4.0",
```
Comment (jaedag) [inline api/package.json:31]: But then why are we dropping the version number if that's. It the solution

**Review by jaedag — CHANGES_REQUESTED** [review]

File: web-react-ts/src/global-utils.ts [hunk 3]
```diff
@@ -200,39 +200,63 @@ interface MemberWithTitle extends Member {
...
+  title: { title: 'Pastor' | 'Reverend' | 'Bishop' }[]
 }
 
 export const getHighestTitle = (member: MemberWithTitle) => {
-  if (!member.titleConnection.edges?.length) {
+  if (!member.title?.length) {
     return
   }
   let highestTitle
+  let power = 0
```
[the remark this answers is not in the document]
Comment (dabick14) [reply web-react-ts/src/global-utils.ts:211]: It's a variable I'm using in the return highestTitle function. The current function isn't working well.

File: api/package.json (hunk 2 above)
Comment (dabick14) [reply api/package.json:31]: Will change it back to 3.4.0

Comment (jaedag) [reply api/package.json:31]: 3.5.0

File: web-react-ts/src/global-utils.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (jaedag) [reply web-react-ts/src/global-utils.ts:211]: Okay. Will call for explanation

File: api/package.json (hunk 2 above)
Comment (dabick14) [reply api/package.json:31]: Ok. Will do

**Review by ahadziii — APPROVED** [review]

**Comment by dabick14** [comment]
@jaedag please can you approve this thing again. Thank you

**Pull request merged**